### DISCUSSION TITLE
NegativeBinomialOutput in DeepAR

### DISCUSSION THREAD

**Discussion by hasiro80** [autogluon/autogluon#2693] [Q&A] [answered]
Hi,
I have an intermittent Time Series and would like to integrate in Hyperparameters distr_output = NegativeBinomialOutput().
How has the Hyperparameter distr_output right set in predictor.fit and which module I need for this?

`predictor.fit(
    train_data,
    hyperparameters={
        "DeepAR": {"distr_output":"NegativeBinomialOutput()"},
    }
)`

Best regards,
Roger

**Comment by shchur** [comment, marked as answer]
For this you need to import `NegativeBinomialOutput` from GluonTS:
```python
from gluonts.torch.distributions import NegativeBinomialOutput

predictor.fit(
    train_data,
    hyperparameters={
        "DeepAR": {"distr_output": NegativeBinomialOutput()}
    },
)

```
